example/lib/styles/text_style.dart

```diff
@@ -1,90 +1,74 @@
 import 'package:flutter/material.dart';
 
 /// TUITextStyle is a class that holds all the text styles used in the Tarka UI.
```

Comment:
@kalpeshp0310 Please change the name in the comment as well.

Reply:
I will remove the comments, as it is in example application and not really part of the library.

Reply:
Agreed it is part of the kitchen-sink app.

Reply:
Done.